src/helpers/formatEpochToDate.ts

```diff
@@ -0,0 +1,17 @@
+export const formatEpochToDate = (epoch: number) => {
+  const date = new Date(epoch * 1000);
+  const dateTimeFormatter = new Intl.DateTimeFormat("en-US", {
+    weekday: "short",
+    month: "short",
+    day: "numeric",
+    year: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+    second: "numeric",
+    hourCycle: "h24",
+    timeZone: "utc",
```

Comment:
assuming it doesn't matter whether this is capitalized or not ([mdn doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat))

Reply:
Yes, both seem to be correct values. Thanks for checking! :raised_hands: